doc/api/assert.markdown

```diff
@@ -2,42 +2,131 @@
 
     Stability: 3 - Locked
 
-This module is used so that Node.js can test itself. It can be accessed with
-`require('assert')`. However, it is recommended that a userland assertion
-library be used instead.
+The `assert` module was originally written and included in Node.js in order
+to facilitate Node.js' own unit tests. While it can be used by JavaScript
+running within Node.js by using `require('assert')`, it is recommended that a
```

Comment:
When that line about userland modules was added, the idea of suggesting a few came up and was rejected on the grounds that inclusion of a module in Node docs suggests endorsement and/or provides the named modules with a serious advantage over omitted modules in the marketplace of ideas.